src/OgRouteGroupResolverBase.php

```diff
@@ -88,6 +88,10 @@ public static function create(ContainerInterface $container, array $configuratio
    *   The entity, or NULL if we are not on a content entity path.
    */
   protected function getContentEntity() {
+    $route = $this->routeMatch->getRouteObject();
+    if (!$route) {
+      return NULL;
+    }
     // Check if we are on a content entity path.
     $path = $this->routeMatch->getRouteObject()->getPath();
```

Comment:
should this not become `$route->getPath()` ?

Reply:
Yes, fixed that